.idea/workspace.xml

```diff
@@ -4,9 +4,13 @@
     <option name="autoReloadType" value="SELECTIVE" />
   </component>
   <component name="ChangeListManager">
+<<<<<<< HEAD
+    <list default="true" id="9231bab6-4c9f-4f82-9de1-33f54cf876a1" name="Changes" comment="" />
+=======
     <list default="true" id="9231bab6-4c9f-4f82-9de1-33f54cf876a1" name="Changes" comment="">
       <change beforePath="$PROJECT_DIR$/app/controllers/api/scores_controller.rb" beforeDir="false" afterPath="$PROJECT_DIR$/app/controllers/api/scores_controller.rb" afterDir="false" />
     </list>
+>>>>>>> 7191ff490d8af58c9bd5eebc978488891ef48b05
```

Comment:
Why are these here? They are git marks that help you identify code source in a git conflict situation.
Anyway, `.idea` files are editor/IDE specific files, so they shouldn't be present in the repository.
It's a little complicated to get rid of them now, but next time, check if you have staged these kind of files and update `.gitignore` to not includem them in commits.